### PR TITLE
Fix build process

### PR DESCRIPTION
**Pull request by simon-id**
### What does this PR do?

This makes the build process actually work as intended

### Motivation

Building process broke on windows because of Visual Studio 2022, and I discovered that we weren't using the intended version of `node-gyp`. Then I had to install a recent version of python to make it work with the never `node-gyp` version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "install": "node bin/appsec-reinstall.js",
-    "rebuild": "node-gyp rebuild",
+    "rebuild": "node ./node_modules/node-gyp/bin/node-gyp.js rebuild",
```
Comment (bengl) [inline package.json:12]: Why is this change necessary?

Comment (simon-id) [reply package.json:12]: As I understand it, npm ships its own (very old) version of `node-gyp` which is used instead of the locally installed one. See https://github.com/nodejs/node-gyp/blob/master/docs/Updating-npm-bundled-node-gyp.md
I haven't checked for sure where the default `node-gyp` binary is from, but what I know is that it runs a very old version when not adding this `./node_modules/...`.
For example in the other (broken) PR: https://github.com/DataDog/dd-native-appsec-js/runs/5499954004?check_suite_focus=true
image
(we want v9)
